docs/sources/clients/promtail/configuration.md

```diff
@@ -1940,6 +1940,9 @@ The optional `limits_config` block configures global limits for this instance of
 # to avoid OOM scenarios.
 # 0 means it is disabled.
 [max_streams: <int> | default = 0]
+
+Maximum log line byte size allowed without dropping. Example: 256kb, 2M. 0 to disable.
```

Comment:
Was this generated by the automation?

Reply:
No. currently automation is supported only for Loki config. Not any of the clients configs are supported (had the same confusion but clarified it with @ssncferreira )